src/main/cljc/jarl/eval.cljc

```diff
@@ -488,7 +488,10 @@
             (do
               (log/tracef "function <%s> threw error: %s" name (ex-message e))
               (if (true? (get state :strict-builtin-errors))
-                (throw e)
+                (let [[rego-type & m] (str/split (ex-message e) #":")
+                      with-name (str/join ": " (apply conj [rego-type name] (mapv str/trim m)))
```

Comment:
I might be reading this wrong. But if an error contains the string `eval_builtin_error: invalid string: "foo : bar:baz"`, won't that get transformed to `eval_builtin_error: <some-builtin>: invalid string: "foo: bar: baz"`, changing parts of the message that should remain intact?

Perhaps we could just strip out the type prefix from the message (it's always included as `::type` on the `ex-info`, right?) and just `(str/join type-from-ex-info ": " name ": " message-without-type-prefix)`?

Reply:
Sounds good 👍  I'll take a look.

Reply:
@johanfylling just pushed - PTAL :)